apps/examples/secure_storage/secure_storage.c

```diff
@@ -0,0 +1,187 @@
+static int DATA_LENGTH = 128;
+
+#ifdef CONFIG_AMEBAD_TRUSTZONE
+static uint32_t SLOT0_START_ADDR = 0x1a000;
+static uint32_t SLOT1_START_ADDR = 0x1d000;
+#endif
+#ifdef CONFIG_AMEBALITE_TRUSTZONE
+static uint32_t SLOT0_START_ADDR = 0x24000;
+static uint32_t SLOT1_START_ADDR = 0x27000;
+#endif
+static uint32_t SLOT_SIZE = 0x1000;
+static uint32_t SLOT_AREA_OFFEST = 0x23000;
+
+/****************************************************************************
+ * secure_storage_main
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int sstorage_main(int argc, char *argv[])
+#endif
+{
+	printf("Start testing SE Secure Storage!!\n");
+	int length = DATA_LENGTH;
+	memset(a,'A', length);
+	a[length] = '\0';
+
+	printf("Start Security init!!\n");
+	sleep(1);
+	security_handle hnd;
+	security_error res = security_init(&hnd);
+	if (res != SECURITY_OK) {
+		dbg("security_init() Fail\n",__LINE__);
+		sleep(1);
+	}
+	printf("=====================%d==================================================\n",__LINE__);
+
+	sleep(1);
+	security_data data = { (void *)a, length};
+	char ss_path[7];
+
+	printf("Start Secure Storage write!!\n");
+	int slot;
+	for (slot = 0; slot < 4; slot++) {
+		snprintf(ss_path, 7, "ss/%d", slot);
+		memset(a,(0xA0 + slot), length);
+		a[length] = '\0';
+		printf("ss_write_secure_storage() Slot: %d\n",slot);
+		sleep(1);
+		res = ss_write_secure_storage(hnd, ss_path, 0, &data);
+		if (res != SECURITY_OK) {
+			dbg("ss_write_secure_storage() Fail\n",__LINE__);
+			sleep(1);
+		}
+	}
+	printf("=====================%d==================================================\n",__LINE__);
+
+	printf("Start Secure Storage read!!\n");
+	sleep(1);
+
+	for (slot = 0; slot < 4; slot++) {
+		snprintf(ss_path, 7, "ss/%d", slot);
+		printf("ss_read_secure_storage() Slot: %d\n",slot);
+		sleep(1);
+		res = ss_read_secure_storage(hnd, ss_path, 0, &data);
+		if (res != SECURITY_OK) {
+			dbg("ss_read_secure_storage() Fail\n",__LINE__);
+			sleep(1);
+		}
+
+		printf("data.length: %d\n",data.length);
+		printf("data.data:\n");
+		memcpy(b, data.data, data.length);
+		for (int i = 0; i < data.length; i++) {
+			printf("%02x ", b[i]);
+			if (!((i+1)%16)) {
+				printf("\n");
+			}
+		}
+	}
+	printf("=====================%d==================================================\n",__LINE__);
+
+	printf("Start Flash Raw data read!!\n");
+	uint32_t address;
+	for (slot = 0; slot < 4; slot++) {
+		/* The Secure Storage Slot Flash Address , 156 = HEADER 4 + DATA_LENGTH 4 + DATA LENGTH + TAG 16 + FOOTER 4 , 1 = en_display*/
+		length = DATA_LENGTH + 4 + 4 + 16 + 4;
+		if (slot == 0) {
+			address = SLOT0_START_ADDR;
+		} else {
+			address = SLOT1_START_ADDR + (SLOT_SIZE * (slot - 1));
+		}
+
+		sleep(1);
+		printf("Read Raw data from Slot: %d, Area A\n",slot);
+		if (!rtl_ss_flash_read(address, length, a, 1)) {
+			dbg("rtl_ss_flash_read() Fail\n",__LINE__);
```

Comment:
Some logs are `printf`, but this is `dbg`.
But I think this is also required for every cases, because it is for failure case.
How do you think?

Reply:
For standatization all dbg will be changed to printf.